en/firefox/browsers/compare/chrome.ftl

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/chrome/
+
+# HTML page title
+compare-chrome-firefox-vs-chrome-which-is = { -brand-name-firefox } vs. { -brand-name-chrome }: Which is better?
+
+# HTML page description
+compare-chrome-we-compare-mozilla-firefox = We compare { -brand-name-mozilla } { -brand-name-firefox } with { -brand-name-google } { -brand-name-chrome } in terms of privacy, utility and portability
+
+# Main title.
+compare-chrome-comparing-firefox-browser = Comparing { -brand-name-firefox-browser } with { -brand-name-google } { -brand-name-chrome }
+
+# Variables:
+#   $attrs (string) - link to https://monitor.firefox.com with additional attributes for analytics
+compare-chrome-at-firefox-we-have-a-huge = At { -brand-name-firefox }, we have a huge number of dedicated users who appreciate our steadfast dedication to online privacy. For example, the latest version of { -brand-name-firefox } includes a feature called Enhanced Tracking Protection (ETP) which is turned on by default for all users worldwide. ETP blocks over 2,000 trackers, including social trackers from companies like { -brand-name-facebook }, { -brand-name-twitter }, and LinkedIn. It also has an integrated feature called <a { $attrs }>{ -brand-name-firefox-monitor }</a> that automatically notifies you if your password has been breached or needs to be updated. In addition to these protections, Firefox's Private Browsing mode automatically deletes your browsing information such as history and cookies, leaving no trace after you finish your session.
```

Comment:
1). Firefox: `Firefox's Private Browsing mode`
2). Anyway to work around possessive use?